virtual_workstation_setup.sh

```diff
@@ -347,11 +306,12 @@ pushd "$workdir"
     sudo chown ubuntu /data/db
     ssh-keyscan github.com >> ~/.ssh/known_hosts 2>&1
 
+    # Do setup_bash first because it affects the environment, so later steps may
+    # depend on having the right PATH or other settings.
     setup_bash
     setup_master
     setup_60
     setup_cr
-    setup_jira_auth
```

Comment:
Was just talking with @IamXander, and it sounds like we don't need this feature at all any more.